cli/command/environment.go

```diff
@@ -166,11 +176,62 @@ func (e *EnvironmentCommand) create(c *cli.Context) error {
 }
 
 func (e *EnvironmentCommand) delete(c *cli.Context) error {
+	if c.Bool("r") || c.Bool("recursive") {
+		environmentID, err := e.entityIDHelper(c, "environment")
+		if err != nil {
+			return err
+		}
+
+		if err := e.deleteDependecies(environmentID); err != nil {
```

Comment:
[nit] I'm not convinced this helper is giving us much of a 'win'.
Personally, I'd either remove it or break it down by type, e.g.
```go
if c.Bool("recursive") {
    if err := e.deleteEnvironmentLoadBalancers(environmentID); err != nil {
        return err
    }

    if err := e.deleteEnvironmentServices(environmentID); err != nil {
        return err
    }

    if err := e.deleteEnvironmentTasks(environmentID); err != nil {
        return err
    }
}
```

Reply:
✅